modules/mobile/spec/requests/mobile/v1/health/immunizations_spec.rb

```diff
@@ -131,26 +133,10 @@
     end
 
     describe 'vaccine group name and manufacturer population' do
-      let(:immunizations_request_non_covid_paginated) do
```

Comment:
One of these was no used. The others were only used once, so I made them inline.